tests/testthat/helper.R

```diff
@@ -4,11 +4,3 @@ requiet <- function(package) {
     require(package, warn.conflicts = FALSE, character.only = TRUE)
   )
 }
-
-# load all hard dependencies
-library(bayestestR)
-library(insight)
-library(datawizard)
-library(effectsize)
-library(parameters)
-library(performance)
```

Comment:
Instead of deleting these, you can replace the `library()` calls with `requiet()` calls, a function which is defined in the same file. This is just to make sure that we don't need to run tests conditionally depending on whether these strong deps are available or not—they always will be, otherwise the package won't be installed.

Reply:
And make sure to use package name as character (e.g. `requiet("insight")`)